tox.ini

```diff
@@ -19,17 +20,26 @@ deps =
     django >= 1.10, <2.0
     pynamodb
     # Python3.5+ only deps
-    py{35,36}: aiohttp >= 2.3.0
+    py{35,36}: aiohttp >= 2.3.0,<3.0.0
     py{35,36}: pytest-aiohttp
     py{35,36}: aiobotocore
 
 commands =
     py{27,34}: coverage run --source aws_xray_sdk -m py.test tests --ignore tests/ext/aiohttp --ignore tests/ext/aiobotocore --ignore tests/test_async_local_storage.py --ignore tests/test_async_recorder.py
-    py{35,36}: coverage run --source aws_xray_sdk -m py.test tests
+    py{35,36}: coverage run --source aws_xray_sdk -m py.test tests --ignore tests/ext/aiohttp/test_client.py
 
 setenv =
     DJANGO_SETTINGS_MODULE = tests.ext.django.app.settings
 
+[testenv:py36-aiohttp3]
```

Comment:
Does this mean the client 3.x support requires Py3.6+?

Reply:
Nops but tox uses 3.5.2 that is not compatible, aiohttp needs at least 3.5.3. Any suggestion?

Reply:
Not sure if I understand this correctly. Tox will use pre-installed interpreter and skip non-exist ones. If your system installs py3.5.2 it will pick that up. If your system doesn't have py3.5 installed it will just skip py35 testing. I'd prefer to add py35 here so it is not mistakenly considered that this new feature only works with py3.6+.